app/Http/Resources/ItemResource.php

```diff
@@ -81,15 +81,24 @@ private function getDescription()
 
     private function getAuthor()
     {
-        if ($this['item']->author_name) {
-            return $this['item']->author_name;
-        }
+        $localAuthoritiesNames = explode(', ', $this['item']->author_name);
+        $webumeniaAuthoritiesNames = collect($this['webumenia_item']->authorities)->map(
+            fn(object $authority) => formatName($authority->name)
+        );
+        $filteredLocalAuthoritiesNames = collect($localAuthoritiesNames)->filter(
+            fn($name) => !$webumeniaAuthoritiesNames->contains($name)
+        );
+
+        $webumeniaAuthoritiesRoles = collect($this['webumenia_item']->authorities)->map(
```

Comment:
no need to call `collect()` again for `$webumeniaAuthorities`
```suggestion
        $webumeniaAuthoritiesRoles = $webumeniaAuthorities->map(
```